app/models/gdpr_request.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class GDPRRequest < ApplicationRecord
```

Comment:
Could we name the model something around hiding data, and the `request_type` is GDPR request?